zaza/functests/deploy.py

```diff
@@ -0,0 +1,90 @@
+import argparse
+import datetime
+import importlib
+import logging
+import os
+import subprocess
+import sys
+import unittest
+import yaml
+
+import juju_wait
+
+BUNDLE_DIR = "./tests/bundles/"
+DEFAULT_TEST_CONFIG = "./tests/tests.yaml"
+
+def deploy_bundle(bundle, model):
+    """Deploy the given bundle file in the specified model
+
+    :param bundle: str Path to bundle file
+    :param model: str Name of model to deploy bundle in
+    """
+    logging.info("Deploying bundle {}".format(bundle))
+    subprocess.check_call(['juju', 'deploy', '-m', model, bundle])
+
+def add_model(model_name):
+    """Add a model with the given name
+
+    :param model: str Name of model to add
+    """
+    logging.info("Adding model {}".format(model_name))
+    subprocess.check_call(['juju', 'add-model', model_name])
```

Comment:
Do we want to do these through subprocess calls or through the actual python libjuju? An example function getting the deployed applications is available at https://github.com/openstack-charmers/zaza/blob/master/zaza/model.py#L8